doc/index.rst

```diff
@@ -34,6 +34,7 @@
     tutorials/plot.rst
     tutorials/text.rst
     tutorials/contour-map.rst
+    tutorials/color-figure.rst
```

Comment:
We need to rename the file. Perhaps `earth-relief.rst`?

Reply:
Changed to `earth-relief.py` and `earth-relief.rst`.